ports/testcontainers-c/vcpkg.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "testcontainers-c",
+  "version-string": "0.1.0",
+  "description": "Testcontainers library for C - C API and shared library",
+  "homepage": "https://github.com/testcontainers/testcontainers-native",
+  "license": "MIT",
+  "dependencies": [
+    {
+      "name": "testcontainers-bridge",
```

Comment:
That part does not work in `vcpkg` runs yet, even if local installation is fine.
Likely, the port files are messed up somehow